packages/legend-application-studio/src/stores/editor/editor-state/element-editor-state/service/ServiceEditorState.ts

```diff
@@ -153,6 +180,43 @@ export class ServiceEditorState extends ElementEditorState {
     this.selectedTab = SERVICE_TAB.TEST;
   }
 
+  get selectedOwnership(): ServiceOwnerOption | undefined {
+    const ownership = this.service.ownership;
+    if (ownership instanceof DeploymentOwnership) {
+      return {
+        label: 'Deployment',
+        value: ServiceOwnershipType.DEPLOYMENT_OWNERSHIP,
+      };
+    } else if (ownership instanceof UserListOwnership) {
+      return {
+        label: 'UserList',
+        value: ServiceOwnershipType.USERLIST_OWNERSHIP,
+      };
+    }
+    return undefined;
+  }
+
+  setSelectedOwnership(o: ServiceOwnerOption): void {
+    let newVal: ServiceOwnership | undefined;
+    switch (o.value) {
+      case ServiceOwnershipType.DEPLOYMENT_OWNERSHIP: {
+        service_setOwnership(
+          this.service,
+          new DeploymentOwnership('', this.service),
+        );
+        break;
+      }
+      case ServiceOwnershipType.USERLIST_OWNERSHIP: {
+        newVal = new UserListOwnership([], this.service);
```

Comment:
we previously added the current user as the first owner if they choose owner list. should we still do this when they switch to userlist ?

Reply:
Changed